### ISSUE TITLE
More details

### ISSUE THREAD

**Issue by wishforgood** [theitzin/FUS3D#2]
Thanks for your brilliant work and I'm very interested in it! Could you please provide more implementation details like training parameters (like learning rate, epochs) and loss function settings you used? I would like to re-implement this work and learn from it.

**Comment by wishforgood** [comment]
Or is there a basic dataset class? The dataset class MIPT officially provides does not seems to fit to your API...
